src/Platform/OpenGL/Texture.cpp

```diff
@@ -43,7 +43,7 @@ namespace MxEngine
         GL_RG8,
         GL_RG16,
         GL_R16F,
-        GL_R32F,
+        GL_R32F_EXT,//cannot use GL_R32F. Why?
```

Comment:
interesting, on what machine it fails?

Reply:
![image](https://github.com/asc-community/MxEngine/assets/53587145/7cf9d156-a567-4807-8b34-f65ba5c1409b)